lib/crossdoc/editor_js_builder.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+require_relative './styler'
+
+module CrossDoc
+  # Used by the Builder to include content from EditorJS
+  class EditorJsBuilder
+    def initialize(container, styles = {})
+      @container = container
+      @styler = Styler.new styles
+    end
+
+    def build(content)
+      content['blocks'].each_with_object(@container) do |block, container|
```

Comment:
what do we gain by passing `@container` here rather than just using `@container` in the block?